packages/compass-generative-ai/src/components/generative-ai-input.tsx

```diff
@@ -260,103 +329,110 @@ function GenerativeAIInput({
   return (
     <div className={containerStyles}>
       <div className={inputBarContainerStyles}>
-        <div className={inputContainerStyles}>
-          <TextInput
-            className={textInputStyles}
-            ref={promptTextInputRef}
-            sizeVariant="small"
-            data-testid="ai-user-text-input"
-            aria-label="Enter a plain text query that the AI will translate into MongoDB query language."
-            placeholder={placeholder}
-            value={aiPromptText}
-            onChange={(evt: React.ChangeEvent<HTMLInputElement>) =>
-              onChangeAIPromptText(evt.currentTarget.value)
-            }
-            onKeyDown={onTextInputKeyDown}
-          />
-          <div className={floatingButtonsContainerStyles}>
-            {isFetching ? (
-              <div className={loaderContainerStyles}>
-                <SpinLoader />
-              </div>
-            ) : showSuccess ? (
-              <div className={loaderContainerStyles}>
-                <Icon
-                  className={
-                    darkMode
-                      ? successIndicatorDarkModeStyles
-                      : successIndicatorLightModeStyles
-                  }
-                  glyph="CheckmarkWithCircle"
-                />
-              </div>
-            ) : (
-              <>
-                {aiPromptText && (
-                  <IconButton
-                    aria-label="Clear prompt"
-                    onClick={() => onChangeAIPromptText('')}
-                    data-testid="ai-text-clear-prompt"
-                  >
-                    <Icon glyph="X" />
-                  </IconButton>
-                )}
-              </>
-            )}
-            <Button
-              size="small"
-              className={cx(
-                generateButtonStyles,
-                !darkMode && generateButtonLightModeStyles
-              )}
-              disabled={!aiPromptText}
-              data-testid="ai-generate-button"
-              onClick={() =>
-                isFetching ? onCancelRequest() : handleSubmit(aiPromptText)
-              }
+        <div className={cx(inputContainerStyles)}>
```

Comment:
This cx() is redundant, isn't it?

Reply:
Yeah, those are left overs after I was experimenting with styles. Will clean up 👍